cps/web.py

```diff
@@ -1057,13 +1057,13 @@ def send_to_kindle(book_id, book_format, convert):
         result = send_mail(book_id, book_format, convert, current_user.kindle_mail, config.config_calibre_dir,
                                   current_user.nickname)
         if result is None:
-            flash(_(u"Book successfully queued for sending to %(kindlemail)s", kindlemail=current_user.kindle_mail),
+            flash(_(u"Success! Selected book has been sent to your Kindle (%(kindlemail)s)", kindlemail=current_user.kindle_mail),
                   category="success")
```

Comment:
Not a good idea, there is a similar sentence in the program, where I already got negative feedback about (#657). Because it does what it says, the book is only queued and if there several other tasks are pending it takes some time to send the book. This should be expressed in the text